openstack/cbr/v3/tags/requests.go

```diff
@@ -0,0 +1,181 @@
+package tags
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+/*
+ShowVaultProjectTag
+Query the set of all tags of the tenant in the specified Region and instance type
+
+@author Aloento
+@since 0.5.17
+@version 0.1.0
+*/
+func ShowVaultProjectTag(client *golangsdk.ServiceClient) (r TagResult) {
+	_, r.Err = client.Get(showVaultProjectTagURL(client), &r.Body, nil)
+	return
+}
+
+// ----------------------------------------------------------------------------
+
+type ActionType string
+
+const (
+	Filter = "filter"
+	Count  = "count"
+)
+
+type CloudType string
+
+const (
+	Public = "public"
+	Hybrid = "hybrid"
+)
+
+type ObjectType string
+
+const (
+	Server = "server"
+	Disk   = "disk"
+)
+
+type ResourceInstancesRequest struct {
+	// Does not contain any of the tags?
+	WithoutAnyTag bool `json:"without_any_tag,omitempty"`
+	// Returns the full amount of data when there are no filter conditions
+	Tags []Tag `json:"tags,omitempty"`
+	// Include any tag
+	TagsAny []Tag `json:"tags_any,omitempty"`
+	// Label not included
+	NotTags []Tag `json:"not_tags,omitempty"`
+	// Does not contain any tags
+	NotTagsAny []Tag `json:"not_tags_any,omitempty"`
+	// Only the op_service permission can use this field as a resource instance filter condition
+	SysTags []Tag `json:"sys_tags,omitempty"`
```

Comment:
if this resources not persisted in response, please remove or comment

Reply:
please check `Table 2-290` in `Cloud Backup and Recovery API Reference_18072022_alifshit`